packages/next/build/webpack-config.ts

```diff
@@ -232,20 +235,22 @@ export default async function getBaseWebpackConfig(
             )
           },
           name(module: { identifier: Function; rawRequest: string }): string {
-            const rawRequest =
-              module.rawRequest &&
-              module.rawRequest.replace(/^@(\w+)[/\\]/, '$1-')
-            if (rawRequest) return rawRequest
```

Comment:
```js
            // avoid relative paths
            if (rawRequest && rawRequest[0] !== '.') {
              return rawRequest;
            }
```

Reply:
How do you feel about just dropping the names all together?

https://github.com/zeit/next.js/pull/8450#discussion_r315975645